vector/src/main/java/im/vector/riotx/features/home/HomeDetailFragment.kt

```diff
@@ -112,7 +111,11 @@ class HomeDetailFragment @Inject constructor(
                                             ?.navigator
                                             ?.requestSessionVerification(requireContext(), newest.deviceId ?: "")
                                 }
-                                dismissedAction = Runnable {}
+                                dismissedAction = Runnable {
+                                    unknownDeviceDetectorSharedViewModel.handle(
+                                            UnknownDeviceDetectorSharedViewModel.Action.IgnoreDevice(newest.deviceId ?: "")
```

Comment:
Maybe do nothing if newest.deviceId is null or empty (which should not happen anyway)

Reply:
Will be reworked in next RP